src/cli/commands/force/lightning/lwc/__tests__/preview.test.ts

```diff
@@ -87,12 +85,54 @@ describe('Preview Tests', () => {
         const logger = new Logger('test-preview');
         setupLogger(logger);
         jest.spyOn(Setup.prototype, 'run').mockImplementation(failedSetupMock);
-        preview.run().catch((error) => {
```

Comment:
The old code in `preview.run()` was using `Promise.all` to execute 2 promises and that would block until both are done executing before returning. I've changed that and am not using `Promise.all` so switched to awaiting in a try-catch block here in order for `run()` method to complete first before checking whether `failedSetupMock` has been called.